third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -362,6 +368,40 @@ impl TestConfig {
                 generate_file_format: true,
                 dump_annotated_targets: true,
             }
+        } else if path.contains("/default-sbc-optimize/") {
```

Comment:
What is SBC?  Presumably not Southwestern Bell Corporation.

Reply:
I have added a comment here. I also explicitly call out SBC to be stackless bytecode in `experiments.rs`.